Header header
string STATUS_SAFE = Safe
string STATUS_UNSAFE = Unsafe
string STATUS_CRASH = Crash
float32 distance
string status
